### PR TITLE
Add NML Android support to the SDK

### PR DESCRIPTION
**Pull request by tourdedave**

### CODE REVIEW

File: pending-changes.yaml [hunk 1]
```diff
@@ -68,23 +71,26 @@ Apollo SDK:
...
 "@applitools/eyes-cypress":
   feature:
     - Support ESM and TypeScript in Applitools' config file
+    - Add NML support for Android
   bug-fix:
     - 
 "@applitools/eyes-storybook":
   feature:
-    - 
+    - Add NML support for Android
```
Comment (kyryloonufriiev) [inline pending-changes.yaml:93]: Doesn't make sense in storybook, cypress, pptr, and playwright

File: js/packages/core/src/utils/extract-broker-url.ts [hunk 2]
```diff
@@ -1,11 +1,16 @@
 import {type Driver} from '@applitools/driver'
 
 export async function extractBrokerUrl(driver: Driver<unknown, unknown, unknown, unknown>): Promise<string> {
-  if (!driver.isIOS) return null
   try {
+    const selector = driver.isIOS
+      ? '//XCUIElementTypeOther[@name="Applitools_View"]'
+      : driver.isAndroid
+      ? "//android.widget.TextView[contains(@content-desc,'Applitools_View')]"
```
Comment (kyryloonufriiev) [inline js/packages/core/src/utils/extract-broker-url.ts:8]: Why `contains`? Isn't it the exact value?

**Review by kyryloonufriiev — APPROVED** [review]
LGTM 🎉 , just two small comments

File: pending-changes.yaml (hunk 1 above)
Comment (tourdedave) [reply pending-changes.yaml:93]: Right. I got a little trigger happy.

File: js/packages/core/src/utils/extract-broker-url.ts (hunk 2 above)
Comment (tourdedave) [reply js/packages/core/src/utils/extract-broker-url.ts:8]: It is. So, this? `//android.widget.TextView[@content-desc="Applitools_View"]`. That appears to be legit based on what I see in the inspector. I'll try it.

**Pull request merged**